src/duration.rs

```diff
@@ -681,81 +892,73 @@ fn time_unit() {
     let third_hour: Duration = -1 * TimeUnit::Hour / 3;
     let sum: Duration = quarter_hour + third_hour;
     let delta = sum.in_unit(TimeUnit::Millisecond).floor()
-        - sum.in_unit(TimeUnit::Second).floor() * TwoFloat::from(1000.0);
-    println!("{:?}", delta * TwoFloat::from(-1) == TwoFloat::from(0));
+        - sum.in_unit(TimeUnit::Second).floor() * 1000.0;
+    println!("{:?}", delta * -1.0 == 0.0);
     assert!((sum.in_unit_f64(TimeUnit::Minute) + 35.0).abs() < EPSILON);
 }
 
 #[test]
 fn duration_print() {
     // Check printing adds precision
     assert_eq!(
-        format!("{}", TimeUnit::Day * 10.0 + TimeUnit::Hour * 5),
-        "10 days 5 h 0 min 0 s"
+        format!("{}", TimeUnit::Day * 10.0 + TimeUnit::Hour * 5).trim(),
```

Comment:
Why `trim()`?

Reply:
Because my Display implementation adds a space at the end of the string and I thought I'd rather trim() the tests than add complexity to the impl just for a single space